src/apis/brain.js

```diff
@@ -409,73 +411,119 @@ export const createAsset = async params =>
           resolve(false);
         })
         .then( async(receipt) => {
-          //TODO add error handling
           if(receipt.status){
-            const response = await axios.post(UPDATE_ASSETS_URL, {
-              assetId: futureAssetId,
-              assetName: assetName,
-              country: country,
-              city: city,
-              collateral: collateralMyb,
-              collateralPercentage,
-            });
-
-            if(fileList.length > 0){
-              let data = new FormData();
-              data.append('assetId', futureAssetId);
-              for(const file of fileList){
-                data.append('file', file.originFileObj);
+            const filesUploaded = fileList.length > 0;
+            const requiredCallsToInternalActions = filesUploaded ? 3 : 2;
```

Comment:
Could we maybe add a const to represent each of these numbers? I've no idea why `3` or `2` is required.

Reply:
so that is the number of calls we have to make successfully in order to show the success message to the user when he lists an asset. If he doesn't upload files it's 2 (call to asset collateral and airtable) otherwise its 3. I've replaced them with consts in a new commit.